packages/grafana-ui/src/components/VizTooltip/VizTooltipColorIndicator.tsx

```diff
@@ -0,0 +1,64 @@
+import { css, cx } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+
+import { ColorIndicator } from './types';
+import { getColorIndicatorClass } from './utils';
+
+interface Props {
+  color: string;
+  colorIndicator: ColorIndicator;
+}
+
+export type ColorIndicatorStyles = ReturnType<typeof getStyles>;
+
+export const VizTooltipColorIndicator = ({ color, colorIndicator = ColorIndicator.value }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <span
+      style={{ backgroundColor: color }}
+      className={cx(styles.colorIndicator, getColorIndicatorClass(colorIndicator, styles))}
+    />
+  );
+};
+
+// @TODO Update classes/add svgs
+const getStyles = (theme: GrafanaTheme2) => ({
+  colorIndicator: css({
+    marginRight: theme.spacing(0.5),
+  }),
+  series: css({
+    width: '14px',
+    height: '4px',
+    borderRadius: theme.shape.radius.pill,
+  }),
+  value: css({
+    width: '12px',
+    height: '12px',
+    borderRadius: theme.shape.radius.default,
+    fontWeight: 500,
+  }),
+  hexagon: css({}),
+  pie_1_4: css({}),
+  pie_2_4: css({}),
+  pie_3_4: css({}),
```

Comment:
What are these css classes placeholders for? 🤔

Reply:
It's for this..

![Screenshot 2023-10-31 at 12 05 12 PM](https://github.com/grafana/grafana/assets/88068998/33e6df60-b03d-4013-a456-83ea5865912e)

Reply:
And not, not much is testable, I just needed it merged to be able to apply styling and test the other tooltips.